code/meta-parent/pom.xml

```diff
@@ -379,7 +379,7 @@
             <dependency>
                 <groupId>io.github.ibuildthecloud</groupId>
                 <artifactId>gdapi-java-server</artifactId>
-                <version>0.10.6</version>
+                <version>0.10.7-SNAPSHOT</version>
```

Comment:
Will update to latest version after a new gdapi-jave-server is released with the appropriate changes.